CODING_STYLE.md

```diff
@@ -135,12 +135,12 @@ enum class Accuracy
 };
 struct MeanSigma
 {
-	float mean;
-	float standardDeviation;
+	float mean = 0.0f;
+	float standardDeviation = 1.0f;
 };
 double const d = 0;
-int i;
-int j;
+int i = 0;
```

Comment:
Actually these are different and *are* in fact required to be zero-initialized anyways - but I'd still just keep it at the very simple rule "just initialize all PODs" instead of arguing when to do it and when not to do it - there's no harm in always doing it.